Marlin/mesh_bed_leveling.h

```diff
@@ -41,6 +41,11 @@
     static FORCE_INLINE float get_probe_y(int8_t i) { return MESH_MIN_Y + (MESH_Y_DIST) * i; }
     void set_z(const int8_t px, const int8_t py, const float z) { z_values[py][px] = z; }
 
+    bool has_mesh() { return TEST(status, 0); }
```

Comment:
It's fine to make sure macros all have parentheses at definition-time. However, since we can't anticipate user error or how macros might be defined in a Makefile, for example, we still require parentheses wherever a macro is used _as a unit_.